sunxspex/emission.py

```diff
@@ -353,67 +360,64 @@ def get_integrand(*, model, electron_energy, photon_energy, eelow, eebrk, eehigh
                          f"'thick-target', 'thin-target'")
 
 
-def gauss_legendre(x1, x2, npoints):
+def _get_integrand(x_log, *, model, electron_dist, photon_energy, z, efd=True):
     """
-    Calculate the positions and weights for a Gauss-Legendre integration scheme.
+    Return the value of the integrand for the thick- or thin-target bremsstrahlung models.
 
     Parameters
     ----------
-    x1 : `numpy.array`
-
-    x2 : `numpy.array`
+    x_log : `numpy.array'
```

Comment:
Is this just `np.log10(electron_dist)`?  If so, then perhaps it's better to just derive it from `electron_dist` rather than potentially having two inconsistent inputs.

Reply:
These are the points the function is evaluated at for the the integration as it's done in log space so are points so it not the same as the `electron_dist`